common/models/project.json

```diff
@@ -3,6 +3,11 @@
   "plural": "projects",
   "base": "PersistedModel",
   "idInjection": true,
+  "mixins": {
+    "DueDate": {
```

Comment:
The name of the mixin is a camelized version of the kebab-case filename.
The object given as a value is the `options` in the (mixin function)[https://github.com/CyrilleHugues/demo-mixin-loopback/pull/1/files#diff-7be13c6a657e5fa55c7c2aed5979be9bR4]